subprojects/docs/src/docs/release/notes.md

```diff
@@ -24,6 +24,15 @@ Think of every feature section as a mini blog post.
   NOTE: Totally fine to just link to an example that show the feature.
 -->
 
+### Deprecation warning improvements
+
+In this release, deprecation warnings are no longer displayed in the build by default. Instead, all deprecation warnings in the build will be collected and a single "summary" will be rendered at the end of the build:
+
+    There're X deprecation warnings, which may break the build in Gradle 5.0. Please run with --warnings=all to see them.
```

Comment:
:o: Instead of `X` I'd use \<number\> to indicate that it is a counter.

Reply:
Done.